CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# unreleased version
```

Comment:
Is there a reason to leave this unreleased? Why not bump the version?

Reply:
No real reason, but I figured we don't need to release it right away, seeing it's not causing any real problems.